rfd/00xx-ec2-tags.md

```diff
@@ -0,0 +1,35 @@
+---
+authors: Andrew Burke (andrew.burke@goteleport.com)
+state: draft
+---
+
+# RFD X - Import EC2 Instance Tags
+
+## What
+
+Teleport nodes running on EC2 instances automatically add instance tags as labels.
+
+### Related issues
+
+- [#11627](https://github.com/gravitational/teleport/issues/11627)
+
+## Why
+
+The current recommended method of [using EC2 tags as Teleport labels](https://goteleport.com/docs/setup/guides/ec2-tags/) requires
+- Creating a custom script to fetch the tags
+- Individually adding each tag as a dynamic label
+- Using the AWS API gateway, where the cost scales with the number of nodes using it
+
+As of January 2022, [instance tags are available via the instance metadata service](https://aws.amazon.com/about-aws/whats-new/2022/01/instance-tags-amazon-ec2-instance-metadata-service/). This will allow Teleport nodes to discover their own instance tags. Unlike the AWS API gateway, instance metadata requests are free and per-instance.
+
+## Details
```

Comment:
Can you also add details about how labels reconciliation will work. For example, what happens if you have statically defined labels also? Can they conflict with AWS tags? How will Teleport reconcile AWS labels between runs (i.e. will it detect that a tag was removed from an instance, for example)?